service/aws.go

```diff
@@ -0,0 +1,118 @@
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2Types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+)
+
+type MachineAwsClient struct {
+	Client *ec2.Client
+	region string
+}
+
+func newMachineAwsClient(accessKeyId string, accessKeySecret string, region string) (MachineAwsClient, error) {
```

Comment:
@XIAOZHUOWU your code is never called outside of this file. It's not integrated into Casvisor. Try to use the Casvisor frontend to list AWS machines to make sure it can use your code